### PR TITLE
Improve mesheryctl pattern list error message.

### PR DESCRIPTION
**Pull request by Ebi27**
**Notes for Reviewers**

This PR fixes the mesheryctl pattern list error response from ;

Error: Get "http://localhost:9081/api/pattern": dial tcp [::1]:9081: connect: connection refused

To
Error: Unable to reach MESHERY server at %s. Verify your environment's readiness for a MESHERY deployment by running `mesheryctl system check`

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @Ebi27. Will you sync your fork/clone/branch in order to ensure that only files that you've changed show in this PR? Find instructions here - https://github.com/meshery/meshery/blob/master/docs/CONTRIBUTING-gitflow.md

File: mesheryctl/internal/cli/root/pattern/list.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ mesheryctl pattern list
 
 		res, err := client.Do(req)
 		if err != nil {
-			return err
+			return errors.Errorf("Unable to reach MESHERY server at %s. Verify your environment's readiness for a MESHERY deployment by running `mesheryctl system check`. ", mctlCfg.GetBaseMesheryURL())
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/pattern/list.go:51]: Only capitalize "M" in Meshery, please.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: mesheryctl/internal/cli/root/pattern/list.go (hunk 1 above)
Comment (Ebi27) [reply mesheryctl/internal/cli/root/pattern/list.go:51]: okay, on it.

**Comment by leecalcote** [comment]
You'll also need to sign your commits - https://github.com/meshery/meshery/pull/6851/checks?check_run_id=10620373572

**Pull request closed without merging**